frontend/src/layout/navigation/navigationLogic.ts

```diff
@@ -153,8 +153,15 @@ export const navigationLogic = kea<navigationLogicType<WarningType>>({
             null as string | null,
             {
                 loadLatestVersion: async () => {
-                    const versions = await api.get('https://update.posthog.com/versions')
-                    return versions[0].version
+                    const versions = (await api.get('https://update.posthog.com/versions')) as VersionType[]
+                    for (const version of versions) {
+                        if (version?.release_date && dayjs(version.release_date) > dayjs()) {
```

Comment:
Nitpick question: Does this cause a release to be delayed a few hours in certain TZs?

Reply:
I think it does, sorry didn't have internet yesterday to check the `dayjs` docs, will update